Add CollectionFS and $meteorCollectionFS Tutorial and Example

dotansimha opened the pull request:
- Add $meteorCollectionFS and CollectionFS tutorial
- Update CollectionFS example.

Urigo commented on examples/collectionfs/.gitignore:
```diff
@@ -1,4 +0,0 @@
-.meteor/local
```
@dotansimha why?

Urigo commented on examples/collectionfs/client/example.js:
```diff
@@ -1,24 +0,0 @@
-angular.module('example', ['angular-meteor']);
```
@dotansimha why did you remove the example and it's tests?

dotansimha commented:
@Urigo 
Fixed! :)

dotansimha commented:
@Urigo , please do not merge yet, missing tutorial for Edit & Crop image

dotansimha commented:
@Urigo 
Done with the tutorial and added all examples! :)

dotansimha commented:
@Urigo , Fixed tests. this caused by Travis that checkout the master for a reason, and then used the PR, so some of my changes was not available in examples/collectionfs/packages/angular-meteor symlink...
Fixed by using the real package of angular-meteor.
